streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDbSessionBytesStoreSupplier.java

```diff
@@ -38,12 +36,14 @@ public String name() {
         return name;
     }
 
+    @SuppressWarnings("deprecation")
     @Override
     public SessionStore<Bytes, byte[]> get() {
-        final RocksDBSegmentedBytesStore segmented = new RocksDBSegmentedBytesStore(name,
-                                                                                    retentionPeriod,
-                                                                                    NUM_SEGMENTS,
-                                                                                    new SessionKeySchema());
+        final RocksDBSegmentedBytesStore segmented = new RocksDBSegmentedBytesStore(
+            name,
+            retentionPeriod,
+            org.apache.kafka.streams.state.internals.RocksDBSessionStoreSupplier.NUM_SEGMENTS,
```

Comment:
Could we use `DEFAULT_NUM_SEGMENTS` from `Windows` instead?

Reply:
Maybe. Not sure. I just tried to get rid of warnings... Don't want to change the actual code. If you think we should consider this, please create a Jira or do a minor PR.

Reply:
Sounds good.